File: pset2/caesar.c
#include<stdio.h>
#include<cs50.h>
#include<ctype.h>
#include<stdlib.h>
#include<string.h>
int main(int argc,string argv[])
{
    string ptext;
    int c;
    if(argc==1||argc>2)
    {
       printf("Usage: ./caesar k\n");
       return 1;
    }
    else
    {
    int k=atoi(argv[1]);
    printf("plaintext: ");
    ptext=get_string();
    int l=strlen(ptext);
    printf("ciphertext: ");
    for(int i=0;i<l;i++)
    {
        if(isalpha(ptext[i]))
        {
          if(ptext[i]>=65&&ptext[i]<=90)
          {
              c=(ptext[i]-65+k)%26+65;
              printf("%c",toupper(c));
          }
          else
          {
              c=(ptext[i]-97+k)%26+65;
              printf("%c",tolower(c));
          }
        }
        else
        printf("%c",ptext[i]);
    }
    printf("\n");
    
        return 0;
    }
 
}

File: pset6/caesar.py
import cs50
import sys
k=int(sys.argv[1])
print('plaintext: ',end='')
ptext=cs50.get_string()
l=len(ptext)
print('ciphertext: ',end='')
for c in ptext:
    if c.isalpha:
        if ord(c)>=65 and ord(c)<=90:
            i=(ord(c)-65+k)%26+65
            print(chr(i),end='')
        else:
            i=(ord(c)-97+k)%26+97
            print(chr(i),end='')
    else:
        print(c,end='')
print()        